packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -796,6 +905,8 @@ abstract class Element implements BuildContext {
     _slot = newSlot;
     _depth = _parent != null ? _parent.depth + 1 : 1;
     _active = true;
+    if (parent != null) // Only assign ownership if the parent is non-null
+      _owner = parent.owner;
```

Comment:
Do we want to assert something about the state of the _owner field if the parent is null?  For example, perhaps its supposed to always be null.  Maybe we should assert that at the entry to this function?